Turn race dialog on (take it out from behind experiment flag)

ewjordan opened the pull request:
Now that the e-mail has gone out to teachers advising them that this dialog will show up, we can take it out from behind the experiment flag.

ashercodeorg commented:
Please determine whether the CircleCI failures are legitimate. Please update any (eyes) tests...there is an eyes test, right?

LGTM.

ashercodeorg commented:
Has there been any manual bashing on `staging` or `test`? It probably is worth a few minutes on a few browsers to confirm manually that data is being stored as expected and that no errors are happening.

ewjordan commented:
Added the eyes test in, and updated it to omit the experiment flag.

The failing CircleCI test was unrelated to anything in this, and yes, I did several rounds of testing in a few different browsers on production (where it's behind a flag) to make sure data was coming through and the dialog was functional.
